src/codecatalyst/activation.ts

```diff
@@ -96,3 +98,35 @@ export async function activate(ctx: ExtContext): Promise<void> {
         authProvider.removeSavedConnection()
     })
 }
+
+async function showReadmeFileOnFirstLoad(workspaceState: vscode.ExtensionContext['workspaceState']): Promise<void> {
+    getLogger().info('codecatalyst: showReadmeFileOnFirstLoad()')
+    // Check dev env state to see if this is the first time the user has connected to a dev env
+    const isFirstLoad = workspaceState.get('aws.codecatalyst.devEnv.isFirstLoad', true)
+
+    if (!isFirstLoad) {
```

Comment:
skip if `isCloud9` ?

Reply:
Before this function is called we exit early if the IDE is not VSCode, but I'll add this check for safety